admin/client/App/sagas/index.js

```diff
@@ -8,8 +8,9 @@ import { loadItems } from '../screens/List/actions';
 /**
  * Load the items
```

Comment:
I *think* that the filter methods actually want to use the `updateParams` method, as that performs a `loadItems` as its last step. No need to run both. in a new saga action.